test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -1804,9 +1825,9 @@ def test_argsort_axis(self):
     @parametrize(
         "a",
         [
-            np.array([0, 1, np.nan], dtype=np.float16),
-            np.array([0, 1, np.nan], dtype=np.float32),
-            np.array([0, 1, np.nan]),
+            subtest(np.array([0, 1, np.nan], dtype=np.float16), name='f16'),
+            subtest(np.array([0, 1, np.nan], dtype=np.float32), name='f32'),
+            subtest(np.array([0, 1, np.nan]), name='default_dtype')
```

Comment:
Why do we keep finding instances of these?

Reply:
We did not run with `np` being numpy before, and names/reprs are slightly different (apparently).